index.js

```diff
@@ -2,6 +2,5 @@
 'use strict';
 
 // Shim for usage with npm/browserify instead of bower
-require('angular-route');
```

Comment:
I don't understand how this line could cause any trouble. Could you please check if we can resolve your issue without this particular change?

Reply:
After I fixed the dedupe issue that line is no longer causing issues